packages/@angular/cli/commands/build.ts

```diff
@@ -25,6 +25,12 @@ export const baseBuildCommandOptions: any = [
   { name: 'locale', type: String },
   { name: 'extract-css', type: Boolean, aliases: ['ec'] },
   {
+    name: 'compress',
+    type: Boolean,
+    default: true,
```

Comment:
> This can be done by adding the option values here: https://github.com/angular/angular-cli/blob/master/packages/%40angular/cli/models/webpack-config.ts#L79

It already is

> The default here can be removed

I disagree, the default value may be redundant here, but is also included in the CLI help output